src/workspaces/container/WorkspaceTabs.ts

```diff
@@ -109,6 +131,7 @@ const getTabs = (workspace?: Workspace): { name: string; link: string }[] => {
       ...commonTabs,
       { name: 'workflows', link: 'workspace-workflows' },
       { name: 'submission history', link: 'workspace-submission-history' },
+      { name: 'settings', link: '#' },
```

Comment:
This is fine as-is and I am giving the PR a :+1:.

Another approach would be to add more configuration here, something like:
```
{ name: 'settings', link: '#', onClick: 'myCustomShowSettingsFunction' },
```
and then instead of checking for `if (tab?.name === 'settings')` or `if (currentTab === 'settings')` you can check if `tab?.onClick` is defined and use its value.

If we anticipated making more changes to the tab bar I think this would be a better approach but ... that's unlikely

Reply:
Done: https://github.com/DataBiosphere/terra-ui/pull/5366/commits/6db4cc5aadda6de9796b7cdbdd6582e814cf6a1e